submit_cycle.sh

```diff
@@ -16,60 +16,137 @@
 # -decide how to manage soil moisture DA. Separate DA script to snow? 
 # -add ensemble options
 
+File_setting=$1
 ############################
-# experiment name 
-
-exp_name=openloop_testing
-
-############################
-# model options
-
-export ensemble_size=1 # ensemble_size of 1 = do not run ensemble 
-                       # LETKF-OI pseudo ensemble uses 1
-
-atmos_forc='gdas' # options: gdas, gswp3, gefs_ens
-
-dates_per_job=2 # number of cycles to submit in a single job
-
-############################
-# DA options
-
-# select YES or NO
-export do_DA=YES  # do full DA update
-do_hofx=NO  # use JEDI to calculate hofx, but do not do update 
-             # only used if do_DA=NO  
-export ASSIM_IMS=NO
-export ASSIM_GHCN=YES
-export ASSIM_SYNTH=NO
+# read in CYCLEDIR and exp_name
+while read line
```

Comment:
I suggest creating a new input variable DIFF_CYCLEDIR to be included in the setting file. Then, if this variable is set, we use it for the CYCLEDIR. If it's not set, we use the current directory. 

Note that with OUTDIR and the executables having a fixed location relative to CYCLEDIR and the current directory we won't need this separate loop.

Reply:
I created the new variable DIFF_CYCLEDIR  and overwrite CYCLEDIR if it is set. We still need this separate loop.

Reply:
Haven't you removed the separate loop?